setup.py

```diff
@@ -4,7 +4,7 @@
 
 
 INSTALL_REQUIREMENTS = [
-    "Django>=1.11,<3.0",
+    "Django>=1.11,<4.0",
```

Comment:
Assuming that support for Django 1.11 is being dropped - should this be `"Django>=2.2,<4.0",` ?